products/surface.md

```diff
@@ -12,69 +12,111 @@ releaseColumn: false
 releaseDateColumn: true
 sortReleasesBy: 'release'
 releases:
-  - releaseCycle: Surface Laptop go / Pro X SQ2
-    release: 2020-10-13
-    eol: 2023-10-13
-  - releaseCycle: Surface Book 3 
-    release: 2020-05-26
-    eol: 2024-05-26
-  - releaseCycle: Surface Go 2
-    release: 2020-05-06
-    eol: 2024-05-06
-  - releaseCycle: Surface Pro X
-    release: 2019-11-05
-    eol: 2023-11-05
-  - releaseCycle: Surface Laptop 3 / Pro 7
-    release: 2019-10-22
-    eol: 2023-10-22
-  - releaseCycle: Surface Go with LTE Advanced
-    release: 2018-11-20
-    eol: 2022-11-20
-  - releaseCycle: Surface Laptop 2 / Pro 6
-    release: 2018-10-16
-    eol: 2022-10-16
-  - releaseCycle: Surface Studio 2
-    release: 2018-10-02
-    eol: 2022-10-02
-  - releaseCycle: Surface Go
-    release: 2018-08-02
-    eol: 2022-08-02
-  - releaseCycle: Surface Pro LTE (Model 1807)
-    release: 2017-12-01
-    eol: 2021-12-01
-  - releaseCycle: Surface Book 2
-    release: 2017-11-17
-    eol: 2021-11-17
-  - releaseCycle: Surface Pro (5th gen)
-    release: 2017-06-15
+  - releaseCycle: Surface RT1
```

Comment:
A few rows are incorrectly copied (1 is a footnote marker on the upstream page: https://docs.microsoft.com/en-gb/surface/surface-driver-firmware-lifecycle-support).

Reply:
I like that we're reversing the table in source so it matches upstream and makes for easier contributions 👍🏽, but the sorting feels off. Going to try a few options (We want the most supported options to be near the top ideally)